src/core/Select/Select.tsx

```diff
@@ -196,6 +197,8 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
   const toggle = () => setIsOpen((open) => !open);
 
   const selectRef = React.useRef<HTMLDivElement>(null);
+  const toggleButtonRef = React.useRef<HTMLSpanElement>(null);
+  const inputWithIconRef = React.useRef<HTMLDivElement>(null);
```

Comment:
I see you are using this ref for focusing select-button when icon is clicked, but that doesn't do anything. In ComboBox, the menu automatically opens when the input is focused, but that's not the case here

Reply:
removed